libs/challenge-registry/ui/src/lib/challenge-card/challenge-card.component.scss

```diff
@@ -1,181 +1,74 @@
-  width: 496px;
-  height: 521px;
+.challenge-card {
+  width: 340px;
+  min-height: 360px;
+  padding: 12px;
   display: flex;
   position: relative;
   align-items: flex-start;
   flex-direction: column;
+  border-radius: constants.$dl-radius-radius-radius16;
 }
-.challenge-card-container1 {
-  top: 0px;
-  left: 0px;
-  width: 496px;
-  height: 521px;
-  display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-}
-.challenge-card-card-banner {
-  top: 27px;
-  left: 28px;
-  width: 440px;
-  height: 137px;
+
+// BANNER
+.card-banner {
+  width: 100%;
+  height: 130px;
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
   align-items: flex-start;
-  flex-shrink: 1;
 }
-.challenge-card-text {
-  top: 54px;
-  left: 159px;
-  height: auto;
+.star-btn {
   position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-starred {
-  top: 36px;
-  left: 381px;
-  width: 78px;
-  height: 42px;
+  top: 18px;
+  right: 18px;
+  height: 32px;
+  padding: 0 8px;
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-  flex-shrink: 1;
-}
-.challenge-card-text1 {
-  top: 10px;
-  left: 22px;
-  height: auto;
-  position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-icon {
-  top: 0px;
-  left: 36px;
-  width: 42px;
-  height: 42px;
-  position: absolute;
-  box-sizing: border-box;
-  object-fit: cover;
-  display: inline-flex;
   justify-content: center;
   align-items: center;
+  border-radius: constants.$dl-radius-radius-radius16;
+  cursor: pointer;
+}
+.card-icon {
+  margin: 0 2px;
 }
-.challenge-card-card-title {
-  top: 194px;
-  left: 75px;
-  width: 347px;
-  height: 86px;
+
+// BODY
+.card-body {
+  margin: 8px;
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
+  flex: 1;
   align-items: center;
   flex-direction: column;
-  justify-content: space-between;
-}
-.challenge-card-text2 {
-  width: 347px;
-  height: auto;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-text3 {
-  height: auto;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-text4 {
-  top: 311px;
-  left: 41px;
-  width: 420px;
-  height: auto;
-  position: absolute;
-  align-self: auto;
-  text-align: left;
-  text-decoration: none;
+  justify-content: flex-start;
 }
-.challenge-card-card-footer {
-  top: 456px;
-  left: 38px;
-  width: 420px;
-  height: 41px;
+
+// FOOTER
+.card-footer {
   display: flex;
-  position: absolute;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
   border-top: 1px solid #c4c4c4;
-  align-items: flex-start;
+  align-items: center;
+  height: 50px;
 }
-.challenge-card-status-tag {
-  top: 456px;
-  left: 323px;
-  width: 135px;
-  height: 41px;
+.difficulty-tag {
   display: flex;
-  position: absolute;
+  flex: 1;
+  height: 24px;
+}
+.status-tag {
```

Comment:
suggest to add some height or padding in the status tag, i.e. :
current: <img width="877" alt="Screen Shot 2022-09-23 at 12 15 40 PM" src="https://user-images.githubusercontent.com/73901500/192006350-9f8184c2-9165-4879-bbf9-94a6d62ccf1d.png">
adding `padding: 6px 0;`: <img width="817" alt="Screen Shot 2022-09-23 at 12 15 28 PM" src="https://user-images.githubusercontent.com/73901500/192006416-cd309697-e6be-4670-8567-7a69171a8da0.png">